examples/comp_deps/child/include/empty.hh

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2018 Open Source Robotics Foundation
```

Comment:
This is probably a nit, but should we update the year on these new licenses to 2020? Or does it not really matter?

Reply:
I tried to keep the year of when the files were created (note that some files were renamed in 2019).

Also, if you check the `cc` files for those examples, they have copyrights that say 2018. So I think this is ok.